x-pack/legacy/plugins/canvas/public/legacy_plugin_support.ts

```diff
@@ -40,8 +40,3 @@ export const legacyRegistries = {
   templates: new LegacyRegistry(),
   tagUIs: new LegacyRegistry(),
 };
-
-(global as any).kbnInterpreter = Object.assign(
```

Comment:
This needs to stay.  This is how legacy 3rd party plugins add functionality to Canvas. We want to hold off breaking that until we have a solid migration plan for the community to follow.

Reply:
this is only inside canvas, shouldn't prevent us from deleting the old interpreter plugin right ?

Reply:
I've reverted it.